### DISCUSSION TITLE
Monitor EPS per Node

### DISCUSSION THREAD

**Discussion by ng1991** [Security-Onion-Solutions/securityonion#9543] [Unsupported Versions]
Hi I was wondering if it is possible to somehow monitor the EPS value from the grid page via a tool like PRTG or Zabbix.

Recently we noticed all our nodes had 0 EPS we resolved this and the data started flowing but we were not alerted of any issues so had a period of no data.

We have monitoring on all nodes but all were showing as status 0 or OK, it would be useful to have a way to know when data isn't being processed into elastic so was wondering if there was a way to do this.

**Comment by cm-ops** [comment]
You could set up a Grafana alarm to monitor this https://docs.securityonion.net/en/latest/grafana.html Here is a link to the YouTube video as well https://www.youtube.com/watch?v=8FmZ4MRe8Uk

**Reply by ng1991** [reply]
Thank you will take a look
